packages/legend-studio/src/models/metamodels/pure/action/generation/DatabaseBuilderInput.ts

```diff
@@ -45,26 +47,50 @@ export class StorePattern {
   }
 }
 
-export class GenerateStoreInput {
-  targetPackage = '';
-  targetName = '';
+export class DatabaseBuilderConfig {
```

Comment:
Is this really how we name it in the backend? Why not `GenerateDatabase`?

Reply:
Strange to me that we name things inconsistently `StorePattern` vs `DatabaseBuilder`, etc.